dspace-api/src/main/java/org/dspace/app/mediafilter/ImageMagickThumbnailFilter.java

```diff
@@ -119,6 +122,25 @@ public File getImageFile(File f, int page, boolean verbose)
         f2.deleteOnExit();
         ConvertCmd cmd = new ConvertCmd();
         IMOperation op = new IMOperation();
+
+        // Check the PDF's MediaBox and CropBox to see if they are the same.
+        // If not, then tell ImageMagick to use the CropBox when generating
+        // the thumbnail because the CropBox is generally used to define the
+        // area displayed when a user opens the PDF on a screen, whereas the
+        // MediaBox is used for print. Not all PDFs set these correctly, so
+        // we can use ImageMagick's default behavior unless we see an explit
+        // CropBox. Note: we don't need to do anything special to detect if
+        // the CropBox is missing or empty because pdfbox will set it to the
+        // same size as the MediaBox if it doesn't exist.
+        PDPage pdfPage = PDDocument.load(f).getPage(1);
```

Comment:
We only need to test page one since we are only generating a thumbnail from page one. I'm not sure if we need to add that to the inline comment?

Reply:
There was actually an issue here in that PDDocument's `getPage()` uses a zero-based index, so I was actually checking the *second* page and this was failing on PDFs with only one page!